client/src/components/Pages/Home/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -4,21 +4,213 @@ exports[`components/Pages/<Home /> should render component 1`] = `
 <div
   className="home-page-container"
 >
-  <EmptyView
-    section="home"
+  <div
+    className="basic-info-section section row"
   >
+    <div
+      className="left-column col s12 m6"
+    >
+      <img
+        alt="Me"
+        className="my-profile-photo"
+        src="test-file-stub"
+      />
+    </div>
+    <div
+      className="right-column col s12 m6"
+    >
+      <h4
+        className="name-lastname"
+      >
+        Dmytro Kovalenko
+      </h4>
+      <p
+        className="job-title"
+      >
+        Full Stack JS Developer
+      </p>
+      <p
+        className="location"
+      >
+        <a
+          href="https://www.google.es/maps?q=41.382207,2.140318"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          Sants Estació, Barcelona, Spain
+        </a>
+      </p>
+      <p>
+        <strong>
+          Contact info: 
+        </strong>
+        Contact me in
+         
+        <a
+          href="https://www.linkedin.com/in/kovalenkodmytro/"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          LinkedIn
+        </a>
+      </p>
+      <p>
+        <strong>
+          Age: 
+        </strong>
+        26
+         years old
+      </p>
+      <p>
+        <strong>
+          Civil status: 
+        </strong>
+        Married, no children
+      </p>
+    </div>
+  </div>
+  <div
+    className="summary-section section"
+  >
+    <h3
+      className="section-title"
+    >
+      About me:
+    </h3>
     <p>
-      Welcome to my new personal project!!!
-    </p>
-    <p>
-      Ask questions as you are speaking to me and try to populate the website with the info he will provide.
+      I am a very passionate software developer with a mathematical
+  background who has tried working with many different languages
+  including Pascal, FORTRAN, Python, C/C++, C#, Java and Javascript.
+  Now I am focused on web development with Node and React and I love it!
     </p>
     <p>
-      Every page will have a progress bar which indicates how much info you have already recovered - so you will know where to stop asking about some particular topic.
+      I am a very methodological and disciplined person,
```

Comment:
Maybe this kind of texts also should be mocked, but I prefer to have it in my snapshot in order to see all the changes of the texts.